changelog/README.md

```diff
@@ -182,6 +187,10 @@
 ## 15th January 2024
 * Updated [Request limits](../guidelines/README.md#request-limits) for both Demo and Production environments.
 
+## 8th January 2023

```

Comment:
I don't think we should backdate changes to the docs, that's not being transparent with the user. Instead this should be dated now and a note can be added to say this was a change physically made in the API back in January but omitted from the documentation (BTW it says 2023 but I assume it was 2024?).